lib/tapioca/runtime/reflection.rb

```diff
@@ -174,14 +175,25 @@ def resolve_loc(locations)
         resolved_loc.absolute_path || ""
       end
 
-      sig { params(singleton_class: Module).returns(T.nilable(Module)) }
-      def attached_class_of(singleton_class)
-        # https://stackoverflow.com/a/36622320/98634
-        result = ObjectSpace.each_object(singleton_class).find do |klass|
-          singleton_class_of(T.cast(klass, Module)) == singleton_class
+      # On Ruby 3.2 or newer, Class defines an attached_object method that returns the
+      # attached class of a singleton class without iterating ObjectSpace. On older
+      # versions of Ruby, we fall back to iterating ObjectSpace.
+
+      sig { params(singleton_class: Class).returns(T.nilable(Module)) }
+      if CLASS_ATTACHED_OBJECT # rubocop:disable Sorbet/EmptyLineAfterSig
```

Comment:
Rubocop thinks that this if statement is an empty line after the sig. Is there something I should do to fix this in a more legitimate way? Do you think the cop should be improved to anticipate a case like this?